app/workers/sphinx_indexation_worker.rb

```diff
@@ -19,24 +19,6 @@ class SphinxIndexationWorker < ApplicationJob
   end
 
   def perform(model, id)
-    instance = model.find_by(model.primary_key => id)
-
-    if instance
-      reindex(instance)
-    else
-      delete_from_index(model, id)
-    end
-  end
-
-  protected
-
-  def reindex(instance)
-    ThinkingSphinx::Processor.new(instance: instance).upsert
-  end
-
-  def delete_from_index(model, *ids)
```

Comment:
This method is called from `SphinxAccountIndexationWorker`, line `:18`

Reply:
it was moved to that class
this is also tested btw